tests/rptest/tests/cloud_storage_timing_stress_test.py

```diff
@@ -262,7 +262,8 @@ def __init__(self, test_context):
             cloud_storage_segment_size_min=2 * self.log_segment_size,
             retention_local_target_bytes_default=2 * self.log_segment_size,
             cloud_storage_enable_segment_merging=True,
-            cloud_storage_cache_chunk_size=self.chunk_size)
+            cloud_storage_cache_chunk_size=self.chunk_size,
+            cloud_storage_spillover_manifest_size=None)
```

Comment:
I suspect setting this to none will actually just _not_ write the config, rather than setting it to nullopt.

Reply:
Nevermind. I was looking at the wrong test results :)